File: program/print-fibonacci-series-without-recursion/print-fibonacci-series-without-recursion.r
# take input from the user
nterms =5
# first two terms
n1 = 0
n2 = 1
count = 2
# check if the number of terms is valid
if(nterms <= 0) {
print("Plese enter a positive integer")
} else {
if(nterms == 1) {
print("Fibonacci sequence:")
print(n1)
} else {
print("Fibonacci sequence:")
print(n1)
print(n2)
while(count < nterms) {
nth = n1 + n2
print(nth)
n1 = n2
n2 = nth
count = count + 1
}
}
}

File: program/print-fibonacci-series-without-recursion/print-fibonacci-series-without-recursion.py
terms = int(input('Enter the number of terms:'))
f, s = 0, 1
print('Fibonacci series are:')
print(f, s, end=' ')

while terms-2:
   t = f + s
   f = s
   s = t
   print(t, end=' ')
   terms = terms-1
